ee/clickhouse/queries/trends/clickhouse_trends.py

```diff
@@ -52,6 +53,45 @@ def _run_query(self, filter: Filter, entity: Entity, team_id: int) -> List[Dict[
             serialized_data = self._handle_cumulative(serialized_data)
         return serialized_data
 
+    def _run_query_for_threading(self, result: List, index: int, sql, params):
+        result[index] = sync_execute(sql, params)
+
+    def _run_parallel(self, filter: Filter, team: Team) -> List[Dict[str, Any]]:
```

Comment:
Do we have tests that hit this code sufficiently given that below we have a `if len == 1`?

Reply:
Yep there's a couple in test_trends with multiple events, like `test_per_entity_filtering`